recipes/parsim/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "parsim" %}
+{% set version = "2.0.0.post1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d357a032fe5def94d069748d56083fe16d4089fcdd7f0a93422bd84e19aa1542"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - psm=parsim.cli:psm_command
+    - psm_dakota_driver=parsim.dakota:psm_dakota_driver
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pandas
```

Comment:
Are these requirements needed for building?

Reply:
Removed things not needed for building